lib/twitter_cldr/js/compiler.rb

```diff
@@ -55,6 +56,34 @@ def compile_each(options = {})
           yield file, TwitterCldr.twitter_locale(locale)
         end
       end
+      def compile_test(options = {})
+        options[:minify] = true unless options.include?(:minify)
```

Comment:
Oh, disregard my last comment. I just realized that most of this method comes from the method above. It'd be idea if we could somehow refactor `compile_each` or extract some parts from it that could be reused in `compile_test`, but if you don't see an easy way to do that, I guess we can leave it as is. Just add an empty line above this method (to separate it from the previous one) and remove explicit `return` at the end.